cortex-plugins-core.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "cells-rails", "~> 0.0.6"
   s.add_dependency "cells-haml", "~> 0.0.10"
   s.add_dependency "mimemagic", "~> 0.3.2"
-  s.add_dependency "ckeditor", "~> 4.2.0"
+  s.add_dependency "ckeditor", "= 4.2.0"
 
```

Comment:
Also - you should add the JSON B Accessor gem as a dependency of this gem, since this is where it's being used. As of right now this gem won't work unless they have the JSON B Accessor gem attached to their app, which is bad